packages/components/psammead-section-label/src/titles.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  MEDIA_QUERY_TYPOGRAPHY,
+} from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { getLongPrimer, getDoublePica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+import { getSansBold, getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const minClickableHeightPx = 44;
+const minClickableHeightRem = minClickableHeightPx / 16;
+
+const paddingDir = ({ dir }) => `padding-${dir === 'rtl' ? 'left' : 'right'}`;
+const paddingReverseDir = ({ dir }) =>
+  `padding-${dir === 'rtl' ? 'right' : 'left'}`;
+
+const SectionLabelLink = styled.a.attrs(props => ({
+  'aria-labelledby': props.labelId,
+}))`
+  color: ${C_EBON};
+  text-decoration: none;
+
+  &:focus,
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+SectionLabelLink.propTypes = {
+  href: string.isRequired,
+  labelId: string.isRequired,
+};
+
+const FlexContainer = styled.span`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  min-height: ${minClickableHeightRem}rem;
+
+  align-items: baseline;
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    /* Allows the IndexLinkCta to take up extra vertical space and unsures bar is hidden
+       with very long section titles over 600px */
+    align-items: stretch;
+  }
+`;
+
+const FlexTextContainer = styled(FlexContainer).attrs({
+  role: 'text',
+})``;
+
+const titleMargins = `
+  margin: 1rem 0;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    margin: 0;
+  }
+`;
+
+const Title = styled.span`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)}
+  color: ${C_EBON};
+  background-color: ${C_WHITE};
+
+  ${titleMargins};
+
+  ${paddingDir}: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    ${paddingDir}: ${GEL_SPACING_DBL};
+  }
+
+  /* needed to ensure always vertically centered even when FlexContainer changes alignment */
+  display: flex;
+  align-items: center;
+`;
+
+Title.propTypes = {
+  dir: oneOf(['ltr', 'rtl']).isRequired,
+  id: string.isRequired,
+  script: shape(scriptPropType).isRequired,
+  service: string.isRequired,
+};
+
+const IndexLinkCta = styled.span.attrs({
+  'aria-hidden': 'true',
```

Comment:
Is this needed @aacn500?

Reply:
Discussed offline. We agreed that while not strictly necessary, it is fine to keep to make doubly sure that the CTA is not read out and as documentation-as-code for future developers. Added a note explaining this to the README - see https://github.com/bbc/psammead/pull/1534/commits/171cef31494cb62d8b6e9a3aaf2ee49931ee316f